ts/src/delta.ts

```diff
@@ -195,6 +195,7 @@ export default class delta extends Exchange {
                     'TRC20': 'TRC20(TRON)',
                     'BEP20': 'BEP20(BSC)',
                 },
+                'sandboxMode': false,
```

Comment:
@Dan-krm usually we only do this when there is a specific use case that we have to handle in the sandbox mode (markets loading, signing, headers) etc. Otherwise we let the user use `setSandboxMode`

Reply:
I think we need the other logic in this PR for setting it to sandboxMode to work properly though

Reply:
To use the testnet on delta I need some variation of this in sign:
```
        const sandboxMode = this.safeValue (this.options, 'sandboxMode', false);
        this.setSandboxMode (sandboxMode);
```
Let me know if I'm misunderstanding something